packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -187,6 +208,36 @@ class ServiceExtensionButton {
   PButton button;
 
   void _click() {
+    switch (extensionDescription.extension) {
+      case 'ext.flutter.debugAllowBanner':
+        gaSelect(gaInspector, gaDebugBanner);
+        break;
+      case 'ext.flutter.debugPaint':
+        gaSelect(gaInspector, gaDebugPaint);
+        break;
+      case 'ext.flutter.debugPaintBaselinesEnabled':
+        gaSelect(gaInspector, gaPaintBaseline);
+        break;
+      case 'ext.flutter.showPerformanceOverlay':
+        gaSelect(gaInspector, gaPerformanceOverlay);
+        break;
+      case 'ext.flutter.profileWidgetBuilds':
+        gaSelect(gaInspector, gaTrackRebuilds);
+        break;
+      case 'ext.flutter.repaintRainbow':
+        gaSelect(gaInspector, gaRepaintRainbow);
+        break;
+      case 'ext.flutter.timeDilation':
+        gaSelect(gaInspector, gaSlowAnimation);
+        break;
+      case 'ext.flutter.platformOverride':
+        gaSelect(gaInspector, gaIOS);
```

Comment:
should this send a 'platformOverride' event? It seems like currently we're always sending `iOS`

Reply:
iOS is too close to platform name too, changed to toggleIoS